_pages/en/strategy-build-vs-buy.md

```diff
@@ -37,66 +44,140 @@ th {
+- ESDC Information Management Policy (being drafted)
+- ESDC [Procurement Policies](http://iservice.prv/eng/finance/purchasing/policy.shtml)
+- ESDC Security Policy (being drafted)
+- TBS [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601)
+- TBS [Directive on Security Management](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32611)
+
+### Governance, Compliance, and Reporting
+
+1. IT Solutions are understood as a collection of IT Products and as such, each IT Product must be assessed on its own as well as the sum of all the IT Products composing the IT Solution.
+2. IT Solutions are obtained in a way that minimizes the risks to the department and to its ability to Canadians (see [Risk Managing the Obtention of Software](#risk-managing-the-obtention-of-software)). As such:
+   1. Complex and core to mandate IT Solutions are designed internally to ESDC.
+      - Small and iterative IT Projects are favoured (see [Target Solution Delivery Model](strategy-target-solution-delivery-model.html)) to deliver the IT Products that will be components to the IT Solution.
+      - IT Solutions Architecture is a key partner throughout the lifecycle of the IT Solutions but IT Products teams have authority to operate and take responsibility for their own products.
+      - IT Solutions are architected iteratively over time, with each gate and iteration providing additional information as to choosing the best IT Products to compose them (see [Target Solution Delivery Model](strategy-target-solution-delivery-model.html)).
+      - Leveraging external third party integrators is supported but IT Solutions Architecture remains in control to ESDC throughout its lifecycle and abides by its [Architectural Principles](#architecture).
+   2. Complex but not core to mandate IT Solutions are designed internally.
+   3. Non-complex and not core to mandate IT Solutions may be designed and delivered by third party integrators.
+3. IT Solutions, whether built internally or built by a third party and delivered as a set of pre-packaged COTS to configure and/or as software as a service, adhere to the Architectural principles as listed in section [Architecture](#architecture)
+4. IT Solutions must provide their overall and individual IT products metrics internally to ESDC. Metrics are comprised of the following:
+   - Accessibility test results
+   - Integration testing results
+   - Security test results
+   - API consumption statistics
+   - Production deployment audit traces
+   - Availability statistics
+5. Security Assessment & Authority (SA&A) uses the Target-SA&A methodology
+6. Accessibility Assessment uses the Target-Accessibility methodology
+7. Audit Assessment uses the Target-Audit methodology
+8. IT Solutions reuse existing IT Products, including:
+    - Internally developed software
+    - Open Source Software (internally and externally developed)
+    - Third party Services (SaaS, PaaS, and IaaS) and COTS
+
+### Finance
```

Comment:
Should address the topics of Operational Expense vs Capital Expense here

The Cloud CoE would be a great help as they've been educating finance on the subject for months.